test/markup/css/css_consistency.txt

```diff
@@ -20,3 +20,13 @@ a:visited { color: blue; }
 div::after { content: "test"; }
 div::before { content: open-quote; }
 span:nth-child(33) { color:red; }
+
+div {
+  color: rgb(var(--r), var(--g), var(--b));
+  background: linear-gradient( var(--black), var(--white) );
+  border-color: rgb(
+    var(--r),
+    var(--g),
+    var(--b)
+  );
+}
```

Comment:
Of course you still need to update the expected output (the `expect` file) as well as running `./tools/css` to copy the tests to all 3 of the other CSS like grammars - which may or may not need their own changes to pass.

Reply:
Don&rsquo;t worry. I&rsquo;m on my way.

Reply:
Ah ok. :)

Reply:
![IMG_20201027_172117](https://user-images.githubusercontent.com/1669261/97289477-d21a9900-1879-11eb-93ab-0e62fd72307a.jpg)

Reply:
It's not a node executable... it's bash... you just run it directly. (sorry if I said otherwise before, can't recall)

Reply:
Done.